test/sync/src/androidTest/kotlin/io/realm/kotlin/test/mongodb/shared/SyncConfigTests.kt

```diff
@@ -806,6 +812,79 @@ class SyncConfigTests {
 //        assertNotEquals(factory, configuration2.flowFactory)
 //    }
 
+    @Test
+    fun syncClientResetStrategy_partitionBased() {
```

Comment:
Missing tests for the default values for partition-based and flexible sync, these tests where in AppConfigurationTests before